src/UnitOfWork.php

```diff
@@ -1564,6 +1564,12 @@ static function ($value) {
                     if ($value instanceof BackedEnum) {
                         return $value->value;
                     }
+                    if (!is_scalar($value) || !$value instanceof Stringable) {
+                        throw new UnexpectedValueException(sprintf(
+                            'Unexpected identifier value: Expecting scalar, got %s.',
+                            gettype($value),
```

Comment:
```suggestion
                            get_debug_type($value),
```

Reply:
I didn't know about this one :+1: cheers